src/v5_hal/firmware/include/nodes/LiftNode.h

```diff
@@ -12,11 +12,12 @@
 class LiftNode : public ILiftNode {
 public:
     enum LiftState {
-        UPDATING, HOLDING
+        DOWN, UP_FOR_RINGS, FULLY_UP

```

Comment:
We might also want a "holding" state still, just to keep it at it's current position (like if you manually lift and want it to stay where you go to)